packages/terra-popup/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+
+* Changed
+  * Updated Jest snapshots for terra-popup changes.
```

Comment:
We don't need to mention the changes to snapshots as CHANGELOGs are intended for consumers
```suggestion
```